.github/workflows/ci-build.yaml

```diff
@@ -367,10 +367,13 @@ jobs:
         with:
           name: test-results
           path: test-results
+      - name: combine-go-coverage
+        run: |
+          go tool covdata percent -i=./test-results -o test-results/full-coverage.out
```

Comment:
Sorta closing my eyes and hoping those files made it and are in the test-results dir :). I see we downloaded the test-results dir but I don't see us doing anything with it. We get the `coverage.out` right after and upload that.